ui/redux/selectors/comments.js

```diff
@@ -24,14 +24,16 @@ export const selectCommentsById = (state: State) => selectState(state).commentBy
 export const selectCommentIdsByClaimId = (state: State) => selectState(state).byId;
 export const selectIsFetchingComments = (state: State) => selectState(state).isLoading;
 export const selectIsFetchingCommentsById = (state: State) => selectState(state).isLoadingById;
-export const selectIsFetchingCommentsByParentId = (state: State) => selectState(state).isLoadingByParentId;
 export const selectIsFetchingReacts = (state: State) => selectState(state).isFetchingReacts;
 
 export const selectMyReacts = (state: State) => state.comments.myReactsByCommentId;
 export const selectMyReactsForComment = (state: State, commentIdChannelId: string) => {
   // @commentIdChannelId: Format = 'commentId:MyChannelId'
   return state.comments.myReactsByCommentId && state.comments.myReactsByCommentId[commentIdChannelId];
 };
+export const selectIsFetchingCommentsByParentId = (state: State, parentId: string) => {
+  return selectState(state).isLoadingByParentId[parentId];
+};
```

Comment:
If we want to change the selector, rename to `selectIsFetchingCommentsForParentId`.
`By*` is used to represent the underlying storage, i.e. returning the whole array/map/object.
`For*` represents a specific query.